docs/getting-started.md

```diff
@@ -34,221 +37,379 @@ The high level data flow is as follows:
+
+`parity --config dev --unsafe-expose --jsonrpc-cors="all" --unlock <ETHEREUM_ADDRESS> --password ~/Desktop/password.txt`
+
+The chain should start! It is a chain with 0 block time and instant transactions, making testing very fast. Passing `unsafe-expose` and ` --jsonrpc-cors="all"` as flags allow Metamask to connect. 
+
+This chain will be accessible on `localhost:8545`. All extra information for customization of a parity dev chains is located [here](https://wiki.parity.io/Private-development-chain#customizing-the-development-chain).
+
+Next you need to import the account with the private testnet ether to Metamask. The base account that the normal configuration of parity gives you is `0x00a329c0648769A73afAc7F9381E08FB43dBEA72`. Its private key is:
+
+Private key : 4d5db4107d237df6a3d58ee5f70ae63d73d7658d4026f2eefd2f204c81682cb7 (note this is the private key given along with the parity dev chain, so it is okay to share)
+
+Use Metamask --> import account ---> private key.
+
+You now have an Ethereum account with a ton of ether, and you should be able to set up the migrations on this network and use Truffle. Now, send some ether to the previous account that got created and unlocked, this way you can run `truffle migrations` with this account. Weirdly,  `0x00a329c0648769A73afAc7F9381E08FB43dBEA72` does not get unlocked by parity, probably because it's password is an empty string, and it might not read properly from a blank text file. Anyways, that's why we have to make a new account, and send the test either from the default account. 
+
+#### 4.2.4 Syncing with the Testnet
+
+If you want to sync with the testnet instead of the mainnet, just make sure the local node is a testnet node, or that you are hitting the Infura testnet endpoints. 
+
+## 5 Query the local Graph Node
+With the subgraph deployed to the locally running Graph Node, visit http://127.0.0.1:8000/ to open up a [Graphiql](https://github.com/graphql/graphiql) interface where you can explore the deployed GraphQL API for the subgraph by issuing queries and viewing the schema. 
+
+We have provided a few simple examples below, but please see the [Query API](graphql-api.md#1-queries) for a complete reference on how to query the subgraph's entities.
 
 #### Example
 Query all `Token` entities:
 ```graphql
-query {
+{
   tokens {
     id
-    owner
+    currentOwner
+  }
+}
+```
+Notice that it is just simply `tokens` pluralized, and it will return all of the entites in The Graph. 
+
+Later, when you've deployed the subgraph with this entity, you'll be able to query for a specific value, such as the token ID
+
+```graphql
+{
+  token(id: "c2dac230ed4ced84ad0ca5dfb3ff8592d59cef7ff2983450113d74a47a12") {
+    currentOwner
   }
 }
 ```
 
-## 6 Buidl 🚀
-Start building world-changing dApps on top of your newly deployed GraphQL interface 🗿✨.
+You can also organize the queries. This query below would organize all tokens by their ID, and return the current owner of each token. 
```

Comment:
I find `organize` an odd term here. To me it suggests multiple queries that are somehow organized into e.g. categories.

`organize the queries` -> `sort, filter or paginate query results`?